web/client/plugins/FeatureEditor.jsx

```diff
@@ -269,6 +269,16 @@ const EditorPlugin = compose(
         componentDidMount() {
             // only the passed properties will be picked
             this.props.onMount(pick(this.props, ['showFilteredObject', 'showTimeSync', 'timeSync', 'customEditorsOptions']));
+        },
+        // TODO: fix this in contexts
+        // due to multiple renders of plugins in contexts (one with default props, then with context props)
+        // the options have to be updated when change.
+        componentDidUpdate(oldProps) {
```

Comment:
do we still need the componentDidMount ?

Reply:
yes first execution is component did mount, future calls are made by componentDidUpdate